app/constant.ts

```diff
@@ -87,7 +87,7 @@ export const Azure = {
 
 export const Google = {
   ExampleEndpoint: "https://generativelanguage.googleapis.com/",
-  ChatPath: "v1beta/models/gemini-pro:generateContent",
+  ChatPath: "v1beta/models/gemini-pro:generateContent",  
```

Comment:
Gemini's model is dynamically spliced to the path, and it needs to be modified to be dynamic

Reply:
> Gemini's model is dynamically spliced to the path, and it needs to be modified to be dynamic

so, if I want to use gemini-1.5-pro，i should change the _ChatPath_ as “v1beta/models/models/gemini-1.5-pro:generateContent”?